cli/src/semgrep/commands/ci.py

```diff
@@ -323,6 +325,10 @@ def ci(
             # Run DeepSemgrep when available but only for full scans
             is_full_scan = metadata.merge_base_ref is None
             deep = scan_handler.deepsemgrep if scan_handler and is_full_scan else False
+            SemgrepCore.path()  # initializes a SemgrepCore class-field needed for .deep_path() to work
```

Comment:
What class filed needs to be initialized?

Reply:
Oh never mind, I just mixed things up it seems, it's not needed to call `SemgrepCore.path()` first.